sem_covid/services/pwdb_knn_experiments.py

```diff
@@ -22,75 +21,45 @@
 from sem_covid.services.pwdb_base_experiment import PWDBBaseExperiment
 from sem_covid.services.sc_wrangling.mean_vectorizer import MeanEmbeddingVectorizer
 from sem_covid.services.store_registry import StoreRegistry
+from sem_covid.services.data_registry import LanguageModel
```

Comment:
I am no longer sure that the old ml experiments and what you have developed with KNN and SVM classifiers are necessary any longer. Can you check this with Stefan, please?

Reply:
I deleted them in `feature/sc-335` and it will be no longer in our project after the merge